docs/examples/neofs-ir/control.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This file can be moved to [https://github.com/nspcc-dev/neofs-node/blob/master/config/example/](https://github.com/nspcc-dev/neofs-node/blob/master/config/example/)